### ISSUE TITLE
Test our accuracy again sklearn

### ISSUE THREAD

**Issue by soi** [weberlab-hhu/GeenuFF#190]
Keras may be wrong here:

https://github.com/keras-team/keras/issues/12991
